### PR TITLE
Extract functions and use a loop to be DRYer.

### PR DESCRIPTION
**Pull request by ewollesen**
This script was doing the exact same things for two different k8s
versions. I extracted that behavior into two orthogonal functions,
then used a loop to remove the duplication.

Future improvements would be to modify the generatebundle binary so
that its output is more predictable, and make the versions in the
input files match the values used here.

Edit: As @TerryHowe pointed out, we should also work towards a place where the files present determine the versions of k8s that we're supporting.

*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by ewollesen** [comment]
/hold 
Depends on 212

File: generatebundlefile/hack/generate_bundle.sh [hunk 1]
```diff
@@ -13,39 +14,43 @@
...
+    local version=$1
+
+    cd "${BASE_DIRECTORY}/generatebundlefile/output"
+    awsAuth "$ORAS_BIN" push --username AWS --password-stdin \
+            "${REPO}:v${version}-${CODEBUILD_BUILD_NUMBER}" bundle.yaml
+    awsAuth "$ORAS_BIN" push --username AWS --password-stdin \
+            "${REPO}:v${version}-latest" bundle.yaml
+}
+
+for version in 1-21 1-22; do
```
Comment (TerryHowe) [inline generatebundlefile/hack/generate_bundle.sh:53]: at some point we the k8s versions based on the existence of the various input yaml files

**Review by TerryHowe — APPROVED** [review]
/lgtm
/approve

File: generatebundlefile/hack/generate_bundle.sh (hunk 1 above)
Comment (ewollesen) [reply generatebundlefile/hack/generate_bundle.sh:53]: Oh, yeah, that'd be good too. We should get the files named better so we don't have to tweak things as much too.

**Review by TerryHowe — APPROVED** [review]
/lgtm
/approve

**Pull request merged**